02_cancer_type_classification/lasso_range_analysis/lasso_range_all.ipynb

```diff
@@ -17,6 +17,7 @@
    "source": [
```

Comment:
<p>is it also expected that there is a shift in the distribution from most being near 0 to not most being near 15000?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>That's the result of the larger range of LASSO parameters I described before. It is expected, and I would argue that it makes more sense for the model comparisons since we're covering a more comprehensive range of "overfit" or non-sparse models now.</p>